full-service/src/json_rpc/v1/models/amount.rs

```diff
@@ -60,14 +60,17 @@ impl TryFrom<&MaskedAmount> for mc_transaction_core::MaskedAmount {
             &hex::decode(&src.commitment)
                 .map_err(|err| format!("Could not decode hex for amount commitment: {:?}", err))?,
         );
-        Ok(Self {
+
+        let masked_amount = mc_transaction_core::MaskedAmountV1 {
```

Comment:
This is scary... whats going to happen when we begin supporting masked amount v2? Maybe not in the v1 full-service API?